mne/viz/_3d.py

```diff
@@ -1344,3 +1346,160 @@ def _get_view_to_display_matrix(scene):
                                  [0.,            0.,   1.,        0.],
                                  [0.,            0.,   0.,        1.]])
     return view_to_disp_mat
+
+
+def plot_dipole_mri_orthoview(dipole, trans, subject, subjects_dir=None,
+                              scale_factor=1e9, coord_frame='head', ax=None,
+                              block=False, show=True):
+    """Plot dipoles on top of mri slices in 3-D.
+
+    Browse through the dipoles using mouse scroll or up/down arrows.
+
+    Parameters
+    ----------
+    dipole : instance of mne.Dipole
+        The dipole to plot.
+    trans : dict | str
+        The mri to head trans or path to the transformation file.
+    subject : str
+        The subject name corresponding to FreeSurfer environment variable
+        SUBJECT.
+    subjects_dir : None | str
+        The path to the freesurfer subjects reconstructions. It corresponds to
+        Freesurfer environment variable SUBJECTS_DIR. If None (default),
+        SUBJECTS_DIR is read from environment or config file.
+    scale_factor : float
+        The scaling applied to convert amplitudes to vector (arrow) lengths for
+        the plot. Defaults to 1e9 (nAm -> mm).
+    coord_frame : str
+        Coordinate frame to use, 'head' or 'mri'.
+    ax : instance of matplotlib Axes3D | None
+        Axes to plot into. If None (default), axes will be created.
+    block : bool
+        Whether to halt program execution until the figure is closed. Defaults
+        to False.
+    show : bool
+        Show figure if True. Defaults to True.
+
+    Returns
+    -------
+    fig : instance of matplotlib figure
+        The figure containing 3-D locations of the dipoles.
+    """
+    import matplotlib.pyplot as plt
+    from mpl_toolkits.mplot3d import Axes3D
+    if has_nibabel():
+        import nibabel as nib
+        from nibabel.processing import resample_from_to
+    else:
+        raise ImportError('This function requires nibabel.')
+    if coord_frame not in ['head', 'mri']:
+        raise ValueError("coord_frame must be 'head' or 'mri'. "
+                         "Got %s." % coord_frame)
+    subjects_dir = get_subjects_dir(subjects_dir=subjects_dir,
+                                    raise_error=True)
+    t1_fname = op.join(subjects_dir, subject, 'mri', 'T1.mgz')
+    t1 = nib.load(t1_fname)
+    scale_factor /= np.mean(t1.header.get_zooms())  # voxel to mm
+    ras2vox = t1.header.get_ras2vox()
+    trans = _get_trans(trans, fro='head', to='mri')[0]
+    if coord_frame == 'head':
+        affine_to = trans['trans'].copy()
+        affine_to[:3, 3] *= 1000  # to mm
+        affine_to = np.dot(affine_to, t1.affine)
+        t1 = resample_from_to(t1, (t1.get_shape(), affine_to))
```

Comment:
I think we might *always* need to resample, otherwise we aren't guaranteed that the units are correct -- MRIs don't have to be sampled in 1mm x 1mm x 1mm voxels...

Did you look into the difference between `get_vox2ras` and `get_vox2ras_tkr`? The latter is used in our `setup_volume_source_space` code so I suspect we'll need to use it.